devel/ace/Portfile

```diff
@@ -1,11 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        DOCGroup ACE_TAO ACE+TAO-7_0_5
+github.tarball_from releases
 
 name                ace
 set name_package    ACE
-version             6.5.2
-revision            1
+version             7.0.5
+revision            0
 distname            ${name_package}-${version}
```

Comment:
I think this should not be needed as the github PG should set it automatically, can you try without it?

Reply:
If i dismiss the 'version' from there, the installed package get a very ugly version string because of this project using 'ACE+TAO-7_0_5' as a version, instead of simply "7.0.5". I already tried it with tag prefix 'ACE+TAO-', but the version still contains underscores instead of dots.

Reply:
Fixed in 7513594f3307121f9418fa06d1dc6984b1af42c1.